independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus-extension/code/quarkiverse/java/.github/workflows/release.yml

```diff
@@ -52,35 +52,23 @@ jobs:
           git config --local user.email "action@github.com"
           git config --local user.name "GitHub Action"
 
-      - name: Maven release ${{steps.metadata.outputs.current-version}}
+      - name: Update latest release version in docs
         run: |
-          git checkout -b release
-          mvn -B release:prepare -Prelease -DreleaseVersion=${{steps.metadata.outputs.current-version}} -DdevelopmentVersion=${{steps.metadata.outputs.next-version}}
+          mvn -B -ntp -pl docs -am generate-resources -Denforcer.skip -Dformatter.skip -Dimpsort.skip
```

Comment:
The `copy-resources` in the `docs/pom.xml` also needs to change to the `<phase>generate-resources</phase>`

Reply:
Fixed in [`0996354` (#31046)](https://github.com/quarkusio/quarkus/pull/31046/commits/09963548c62714da1af38cd2a01109d65fecf843)